appendices/review-template.md

```diff
@@ -0,0 +1,94 @@
+## Peer Review Template
+
+```
+## Package Review
+
+*Please check off boxes as applicable, and elaborate in comments below. Your review is not limited to these topics, as described in the reviewer guide*
+
+- [ ] As the reviewer I confirm that there are no conflicts of interest for me to review this work (If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review).
+
+#### Documentation
+
+The package includes all the following forms of documentation:
+
+- [ ] **A statement of need** clearly stating problems the software is designed to solve and its target audience in README
+- [ ] **Installation instructions:** for the development version of package and any non-standard dependencies in README
+- [ ] **Vignette(s)** demonstrating major functionality that runs successfully locally
```

Comment:
@arianesasso yes that word is specific to the R community. In fact if you package an R package there is a vignette function. i'm so used to using it from years of working with R but i wonder if we just want to use short tutorials for Python as most are familiar with it to avoid cognitive overload? i'm somewhat torn on it but most people i meet who don't use python don't know that word?

Reply:
Heey, thanks for clarifying 😄 . I would personally prefer something more general, as you said: short tutorials because then the R community and others could also understand. You could mention (if necessary) something like: short tutorials (for R users, something like vignettes).